rise-storage.html

```diff
@@ -669,7 +669,11 @@
           }
 
           if ( this._isCacheRunning ) {
-            url = this._baseCacheUrl + "/metadata/?url=" + encodeURIComponent( url );
+            if ( this._compareVersionNumbers( this._rcVersionNumber, this._RC_VERSION_WITH_ENCODE ) >= 0 ) {
+              url = this._baseCacheUrl + "/metadata/?url=" + url;
```

Comment:
@rodrigopavezi This is comparing against RC version `1.7.1`, should it be?

Reply:
@stulees good catch, I think we can bump to 1.7.3 because 95% of the players have upgraded to that. Thoughts? cheers

Reply:
@rodrigopavezi Okay agreed, but just for this metadata function, correct? The other comparison stays at `1.7.1`, do I have that right?

Reply:
@stulees I would say it can be for both. Because we have already almost everyone on the 1.7.3. cheers